xshop/invoices/api/views.py

```diff
@@ -0,0 +1,43 @@
+from rest_framework.response import Response
+from rest_framework.views import APIView
+from rest_framework import status
+from ..models import Invoice
+from .serializers import InvoiceSerializer
+
+
+class InvoiceListCreateApi(APIView):
+    serializer_class = InvoiceSerializer
+
+    def get(self, request):
```

Comment:
you need to add  @swagger_auto_schema to your endpoints to make it easier with listing schema and responses sent after each operation to list all those data in swagger-ui 
you can take a look here: https://drf-yasg.readthedocs.io/en/stable/custom_spec.html
and check 74_swagger_update branch

Reply:
This PR shows how #74